.github/labeler.yaml

```diff
@@ -0,0 +1,36 @@
+area/docs:
+- "docs/**/*"
+- "**/*.md"
+
+area/config:
+- "config/*"
+- "config/**/*"
+
+area/installer:
+- "agent-installer/**/*"
+
+area/apis:
+- "apis/*"
+- "apis/**/*"
+
+area/test-release:
+- "hack/**/*"
+- "test/**/*"
+- "go.mod"
+- "Makefile"
```

Comment:
Suggestion: Do we require this label as well?

Reply:
Not super important for now but this will help us identify and filter PR's which are related to either releases or testing. We can later assign people who are experts in that area. Any changes to `go.mod` will directly affect the release and it's important to label them so we can filter and take actions accordingly.

Reply:
+1 to keeping this label.